requirements.txt

```diff
@@ -1,2 +1,4 @@
+colorama==0.3.9
```

Comment:
These requirements are installed inside the docker image, and `validate.py` is run outside of it. We can either add a separate requirements file to install dependencies for the outer script, or we could run `validate.py` inside Docker.

Reply:
Are there any downsides to running it inside Docker? We'd still be able to run it outside manually on arbitrary result files I presume.